docs/pages/machine-id/deployment.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Deploy Machine ID
+description: Explains how to deploy Machine ID on your platform and join it to your Teleport cluster.
+tocDepth: 3
+---
+
+The first step to set up Machine ID is to deploy the `tbot` binary and join a
+Machine ID bot to your Teleport cluster. You can run the `tbot` binary on a
+number of platforms, from AWS and GitHub Actions to a generic Linux server or
+Kubernetes cluster. This guide shows you how to deploy Machine ID on your
+infrastructure.
+
+## Choosing a deployment method
+
+There are two considerations to make when determing how to deploy Machine ID on
+your infrastructure.
+
+### Your infrastructure
+
+The `tbot` binary runs as a container or on a Linux virtual machine. If you run
+`tbot` on GitHub Actions, you can use a ready-made [GitHub Actions
+workflow](https://github.com/marketplace/actions/teleport-auth).
```

Comment:
We might want to avoid linking to a specific action here - since this one only works for SSH and we have a few others.